src/enums/sizes.md

```diff
@@ -3,13 +3,14 @@
 Rust enums are packed tightly, taking constraints due to alignment into account:
 
 ```rust,editable
-use std::mem::{align_of, size_of};
-
-macro_rules! dbg_size {
-    ($t:ty) => {
-        println!("{}: size {} bytes, align: {} bytes",
-                 stringify!($t), size_of::<$t>(), align_of::<$t>());
-    };
+use std::{
+    any::type_name,
+    mem::{align_of, size_of},
+};
```

Comment:
We try to use module-level imports (but I haven't figured out how to run `rustfmt` on inline code snippets yet):

```suggestion
use std::any::type_name;
use std::mem::{align_of, size_of};
```